src/main/resources/data-use-v3.json

```diff
@@ -0,0 +1,69 @@
+{
+  "id": "https://consent-ontology.dsde-prod.broadinstitute.org/schemas/data-use",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Use Schema V3",
+  "version": 2,
+  "type": "object",
+  "dependencies": {
+    "otherRestrictions": {
+      "anyOf": [
+        {"required": ["ethicsApprovalRequired"]},
+        {"required": ["collaboratorRequired"]},
+        {"required": ["geographicalRestrictions"]},
+        {"required": ["other"]}
+      ]
+    }
+  },
+  "properties": {
+    "generalUse": {
+      "type": "boolean"
+    },
+    "diseaseRestrictions": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "uniqueItems": true
+      }
+    },
+    "hmbResearch": {
+      "type": "boolean"
+    },
+    "populationOriginsAncestry": {
+      "type": "boolean"
+    },
+    "methodsResearch": {
+      "type": "boolean"
+    },
+    "commercialUse": {
+      "type": "boolean"
+    },
+    "NPU": {
+      "type": "boolean"
+    },
```

Comment:
Super minor nit, we should probably case this similarly to the rest of the fields.